src/param.F

```diff
@@ -297,7 +297,7 @@ subroutine param(namelist_input,                                  &
       call bcast_int(npt)
       call bcast_int(pdtra)
       call bcast_int(iprcl)
-      call bcast_int(nparcels) ! DEVICE
+      call bcast_int(nparcelsInit) ! DEVICE
```

Comment:
@sjsprecious Where is the corresponding bcast_init() call for nparcelsMax?

Reply:
@johnmauff thanks for catching this! Just added it.